terraform/modules/len/main.tf

```diff
@@ -42,9 +42,14 @@ resource "aws_iam_role" "len" {
   assume_role_policy = data.aws_iam_policy_document.len_assume_policy.json
 }
 
+resource "aws_iam_role_policy_attachment" "len_xray_access" {
+  role       = aws_iam_role.len.name
+  policy_arn = "arn:aws:iam::aws:policy/AWSXRayDaemonWriteAccess"
+}
+
 resource "aws_cloudwatch_event_rule" "schedule" {
-  name                = "schedule"
-  description         = "Schedule for Lambda Function"
+  name                = "LENLambdaSchedule"
+  description         = "Schedule for Lambda Function to send LEN events"
```

Comment:
We're going to have a few of these and they're pretty hard to distinguish in the console. Open to better naming suggestions @oskarwilliams378

Reply:
Can we include at least the environment name in the description if we've got it?